bin/load-mocks

```diff
@@ -153,7 +158,10 @@ def create_system_time_series():
 
 
 def create_sample_time_series(event, release=None):
-    group = event.group
+    try:
+        group = event.group
+    except AttributeError:
```

Comment:
is event `None` in this case? because if so, i think checking `if event is None` and then returning would be better/avoid catching too broad of an error. alternatively, you could also check that before calling this.